base/compiler/tfuncs.jl

```diff
@@ -10,7 +10,7 @@ const _NAMEDTUPLE_NAME = NamedTuple.body.body.name
 
 const INT_INF = typemax(Int) # integer infinity
 
-const N_IFUNC = reinterpret(Int32, arraylen) + 1
```

Comment:
is it intentional that this line was removed?

Reply:
It wasn't removed, it was just modified (look one line down).